src/searchPage/searchPageController.js

```diff
@@ -0,0 +1,6 @@
+import angular from 'angular';
+
+export default angular.module('searchPage')
+    .controller('SearchPageController', ['$translate', function ($translate) {
```

Comment:
What is the point of this controller?

Reply:
the basic usage of $translate service on a controller (or service) level